database/postgresql/postgresql_user.py

```diff
@@ -274,8 +282,9 @@ def user_alter(cursor, module, user, password, role_attr_flags, encrypted, expir
                     role_attr_flags_changing = True
 
         expires_changing = (expires is not None and expires == current_roles_attrs['rol_valid_until'])
+        connection_limit_changing = (connection_limit is not None and connection_limit == current_roles_attrs['rol_conn_limit'])
```

Comment:
Role should be singular.

```current_role_attrs```

This calls out that line 284 is actually broken in its existing state.